module/documents/actor.mjs

```diff
@@ -25,6 +25,17 @@ export default class ActorEd extends Actor {
     this.update( {"system.karma.value": this.system.karma.max} );
   }
 
+  /**
```

Comment:
Why is this in the actor document class? this is for the presentation layer und should be part of the sheet class.
And what does it actually do/where is it called?

Reply:
expand in application und enrichment in document.